bin/ddg2logo

```diff
@@ -107,6 +122,76 @@ def plot_matrix(mat, reslist, letters, xlabels, max_range, title=None, fontsize=
         # AA mutations sorted by energy
         s_pos_idx = pos_idx[0][local_pos_idx]
 
+        
+
+        #if the config file is provided, load the config 
+        if config_file is not None:
+            with open(options.config_file) as f:
+                config = yaml.load(f, Loader=SafeLoader)
+
```

Comment:
I'm not reviewing this part for now - see general comment

Reply:
ok